grid/payments/create-product/profile.supr

```diff
@@ -0,0 +1,76 @@
+name = "payments/create-product"
+version = "1.0.0"
+
+"""
+Create product  usecase
+"""
+
+usecase CreateProduct unsafe {
+  input {
+    """
+    Name
+    Name of the product to be created.
+    """
+    name! string!
+
+    """
+    Description
+    Description of the product to be created.
+    """
+    description! string!
+
+    """
+    Product type
+    Indicates whether the product is physical or intangible (e.g. a service, or digital goods).
+
+    Possible values:
+      - `physical`
+      - `digital`
+      - `service`
+    """
+    type enum {
+      physical,
+      digital,
+      service
+    }
+  }
+
+  result {
+    """
+    ID
+    Unique identifier of the product.
+    """
+    id! string!
+    """
+    Product URL
+    A URL of the product's publicly-accessible webpage.
+    """
+    url string
```

Comment:
question: Where is this exactly used? Isn't this something what should be provided on input?